conf/messages.en

```diff
@@ -362,9 +362,9 @@ removeMemberDetails.title = Are you sure you want to remove this member?
 removeMemberDetails.heading = Are you sure you want to remove {0}?
 removeMemberDetails.error.required = Select yes if you are sure you want to remove
 
-activeAccountDetails.title = Did the scheme have at least 1 active bank account during the period of this return?
```

Comment:
Unfortunately, QA's acceptance tests will fail if we change titles.
So we would need to ask QAs to change their tests too. It's a similarly simple change on their end, but it will fail if we just merge this in.

Reply:
All fixed now

Reply:
Thanks @BabuRupoll7623 , In that case, @joshhumphreys please resolve the conflicts and I'll merge this